src/components/VideoChatButtonAndMenu.js

```diff
@@ -21,11 +21,13 @@ import ROUTES from '../ROUTES';
 const propTypes = {
     ...withLocalizePropTypes,
     ...windowDimensionsPropTypes,
-    isConcierge: PropTypes.bool,
+    openInboxCall: PropTypes.bool,
```

Comment:
Ah yes, before all the design changes, the new button _also_ used this flow, so the variable name was changed. Since the design changes mean they are two different flows, I will change this back! Good catch